src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
thumbsup meansd keep the change or remove it? because this change is tied with the below function which we are talking abt also.